plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPlugin.groovy

```diff
@@ -69,12 +62,18 @@ class AndroidCommandPlugin implements Plugin<Project> {
             factory.create(variant, new InstallExtension('device', 'installs the APK on the specified device'))
         }
     }
-    
-    static defaultTask(Project project, String taskName, String description, Class<? extends AdbTask> taskType, Closure configuration) {
-        AdbTask task = project.tasks.create(taskName, taskType)
-        task.configure configuration
-        task.group = TASK_GROUP
-        task.description = description
+
+    private void configureSystemAnimations(Project project) {
```

Comment:
Removed `defaultTask` method. Instead we create the task ourselves. It is simple actually.